book/team.yaml

```diff
@@ -512,7 +512,7 @@ people:
     user_groups:
       - Tutorial Lead
   # =============
-  - title: Xiaolan Xu	
+  - title: Xiaolan Xu
```

Comment:
@jomey note this `-->` special character doesn't show up in the GitHub editor :(  would be nice to setup precommit to potential avoid some of the headaches around this yaml indent and formatting (https://github.com/uwhackweek/jupyterbook-template/issues/24)

Reply:
As noted in the issue, an `.editorconfig` is perfect and designed for these kind of challenges.

Reply:
@xlxu220 out of curiosity what text editor are you using to make changes locally?

Reply:
It's weird. I just use the mac TextEdit. I copy one of the previous people as a template and edit my own content to make sure of the format. However, it seems a lot of things changed.